distributedlog-core/src/main/java/org/apache/distributedlog/tools/DistributedLogTool.java

```diff
@@ -2854,7 +2856,7 @@ public DistributedLogTool() {
         addCommand(new ListCommand());
         addCommand(new ReadLastConfirmedCommand());
         addCommand(new ReadEntriesCommand());
-        addCommand(new RecoverCommand());
+        // addCommand(new RecoverCommand());
```

Comment:
Thanks. https://github.com/apache/distributedlog/issues/150 is opened to track this issue, will add a comment here in the code.